lib/perlfaq5.pod

```diff
@@ -734,8 +734,7 @@ commas:
 L<CLDR::Number> has many more advanced features to handle just about
 anything that you might want. Olaf Alders' article "Stop Writing Your Own
 Commify Functions" has more examples
-(L<https://www.olafalders.com/2015/09/04/stop-writing-your-own-commify-
-functions/>).
+(L<https://www.olafalders.com/2015/09/04/stop-writing-your-own-commify-functions/>).
```

Comment:
I suspect the author of this section inadvertently applied a wordwrap to their edits and didn't notice it broke the link.

Reply:
Thank you for the comments. This fix seems to now have been merged in PR #144.